### PR TITLE
Gif

### PR DESCRIPTION
**Pull request by ChristianPerez34**
Working on modules that create animations. Started Working with simple .gif files

### CODE REVIEW

**Comment by TheParodicts** [comment]
Looks good; used as basis for videoClip class

**Review by TheParodicts — APPROVED** [review]

File: main.py [hunk 1]
```diff
@@ -8,8 +8,9 @@ def main():
     gif = Gif()
     gif.make_gif(clip=clip, start_time=(0, 24), end_time=(0, 27))
     gif.resize(new_size=0.6)
-    # gif.clip = gif.clip.crop(x1=60, x2=300)
-    gif.clip.write_gif('test1.gif')
+    gif.add_text(text='In my nightmares\nI see rabbits.', font_size=30, color='white',
```
Comment (alejoreyes96) [inline main.py:12]: Can a gif be written without adding any text? I'm guessing you can, but should be added as tests.

**Review by alejoreyes96 — APPROVED** [review]
Everything looks good, clean and commented. Easy to read into.

**Review by lexdyel-mendez — APPROVED** [review]
Everything looks good.

File: main.py (hunk 1 above)
Comment (ChristianPerez34) [reply main.py:12]: Yes.

**Pull request merged**